src/icons/hicolor/scalable/apps/yast-geo-cluster.svg

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A geo cluster means you have two servers at two geographically different places. So I'd put two "pin" icons (or how is that called) there, one for each "server".

Reply:
I don't think there is enough space for two pins

Reply:
OK, so let's use this one.